app/src/components/pages/UploadDocumentsPage/SelectStage.tsx

```diff
@@ -82,66 +81,98 @@ function SelectStage({
       inputRef.current.files = toFileList(updatedValues);
     }
   };
+
+  const mockPatientDetails: PatientDetails = {
```

Comment:
Could this mocked value not come from a service so that we don't have to change this later down the line?

Reply:
Getting patient details is part of story-48, we didn't want to repeat code.